twindb_backup/__init__.py

```diff
@@ -168,3 +169,25 @@ def get_timeout(run_type):
         'yearly': 365 * 24 * 3600 / 2
     }
     return timeouts[run_type]
+
+
+def save_measures(start_time, end_time, log_path=LOG_FILE):
+    """Save backup measures to log file"""
+    data = {}
+    data['start'] = start_time
+    data['finish'] = end_time
+    data['duration'] = end_time - start_time
+    if os.path.isfile(log_path):
+        with open(log_path) as data_fp:
```

Comment:
I'm not sure, but if I remember correctly open() in 'a+' mode will create a file if it doesn't exist. So you can get rid of `if...else`.

Reply:
'a+' is only create file if file doesn't exist or append to file and open return last position in file, that will be empty for json loads.